src/util/colorScale.js

```diff
@@ -203,7 +203,8 @@ function createOrdinalScale(colorBy, t1nodes, t2nodes) {
       const spread = minMax[1] - minMax[0];
       const dp = spread > 5 ? 2 : 3;
       legendValues = genericDomain.map((d) => parseFloat((minMax[0] + d*spread).toFixed(dp)));
-      if (legendValues[0] === -0) legendValues[0] = 0; /* hack to avoid bugs */
+      // Hack to avoid a bug: https://github.com/nextstrain/auspice/issues/540
+      if (Object.is(legendValues[0], -0)) legendValues[0] = 0;
```

Comment:
Haven't actually tried this out, but interestingly the docs say

> The only difference between Object.is() and === is in their treatment of signed zeros and NaN values. The === operator ...  treats the number values -0 and +0 as equal, but treats [NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) as not equal to each other.

So I don't see how the original hack worked in practice. Anyway ,as long as the 🐛  doesn't return this change is great.

Reply:
AIUC, the original `===` comparison meant that both +0 and -0 values got reset to +0, where with the `Object.is` comparison, only -0 values will get reset to +0. The outcome is the same, though (assuming no side effects in the property setter).

Reply:
I assume the reason for the bug is that something else using `legendValues[0]` treated `-0` as different from `+0`, when they should be treated the same. So, as @tsibley points out, either `===` or `Object.is()` can be used to correct that behavior, but the former is discouraged by [no-compare-neg-zero](https://eslint.org/docs/latest/rules/no-compare-neg-zero).